src/test/java/org/folio/oaipmh/helpers/StorageHelperTest.java

```diff
@@ -1,250 +1,166 @@
+    assertThat(getStorageHelper(SOURCE_RECORD_STORAGE).getLastModifiedDate(item), is(notNullValue()));
+  }
+
+  @Test
+  void epochDate() {
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE) + "/instance_withoutMetadata.json");
+    assertThat(getStorageHelper(SOURCE_RECORD_STORAGE).getLastModifiedDate(item), is(Instant.EPOCH));
+  }
+
+  @Test
+  void getItemId() {
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE) + "/instance.json");
+    assertThat(getStorageHelper(SOURCE_RECORD_STORAGE).getRecordId(item), not(isEmptyOrNullString()));
+  }
+
+  @Test
+  void getSuppressedFromDiscovery() {
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE) + "/instance.json");
+    assertFalse(getStorageHelper(SOURCE_RECORD_STORAGE).getSuppressedFromDiscovery(item));
+  }
+
+  @Test
+  void getRecordSource() {
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE) + "/instance.json");
+    String recordSource = getStorageHelper(SOURCE_RECORD_STORAGE).getRecordSource(item);
+    assertThat(recordSource, is(notNullValue()));
+  }
+
+  @Test
+  void shouldReturnLinkedToRecordInstanceId_whenGetIdentifierAndStorageIsSRS(){
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE)+"/instance.json");
+    assertEquals(INSTANCE_ID, getStorageHelper(SOURCE_RECORD_STORAGE).getIdentifierId(item));
+  }
+
+  @Test
+  void shouldReturnEmptyString_whenGetIdentifierIdAndStorageIsSRSAndRecordHasNotExternalIdsHolderField(){
+    JsonObject item = getJsonObjectFromFile(getDirPath(SOURCE_RECORD_STORAGE)+"/instance_withoutExternalIdsHolderField.json");
+    assertEquals(EMPTY, getStorageHelper(SOURCE_RECORD_STORAGE).getIdentifierId(item));
+  }
+
+
+  @Test
+  void getId() {
+    JsonObject item = getJsonObjectFromFile(getDirPathWithPathRecordsInSRS(SOURCE_RECORD_STORAGE) + "/instance.json");
+    assertThat(getStorageHelper(SOURCE_RECORD_STORAGE).getId(item), not(isEmptyOrNullString()));
+  }
+
+  @Test
+  void getRecordsItems() {
+    JsonObject entries = getJsonObjectFromFile(getDirPathWithPathRecordsInSRS(SOURCE_RECORD_STORAGE) + "/marc-e567b8e2-a45b-45f1-a85a-6b6312bdf4d8.json");
+    JsonArray items = getStorageHelper(SOURCE_RECORD_STORAGE).getRecordsItems(entries);
+    assertThat(items, is(notNullValue()));
+    assertThat(items, is(iterableWithSize(1)));
+  }
+
+  private StorageHelper getStorageHelper(String storageType) {
+    System.setProperty(REPOSITORY_STORAGE, storageType);
+    return StorageHelper.getInstance();
+  }
+
+  private String getDirPath(String storageType) {
+    return SOURCE_RECORD_STORAGE.equals(storageType) ? SOURCE_STORAGE_RECORD_PATH : "TODO CHANGE";
+  }
+
+  private String getDirPathWithPathRecordsInSRS(String storageType) {
+    return SOURCE_RECORD_STORAGE.equals(storageType) ? STORAGE_RECORD_PATH : "TODO CHANGE";
```

Comment:
I **_think_** TODO CHANGE here suggests that you plan on cleaning up the ability to specify the storage type (see earlier comment)... +1

Reply:
Well, not it is not possible for end-user to specify the storage type, we will be adding this feature in Q3. Sorry for not cleaning this up.